README.md

```diff
@@ -2,23 +2,142 @@
+This subset of LinkML should be easy to get started with.
+
+#### The 10-minute Intro
+
+The data model primarily models different types of biological data and patients/samples and uses templates that collects information for these entities. 
+(It may helpful to also read about [Entity-Relation Model](https://en.wikipedia.org/wiki/Entity%E2%80%93relationship_model).)
+
+To do this the building blocks are:
+
+| LinkML term        | Schematic Note | Other Note |
+| ------------------ | -------------  |------------|
+| Class              | Usually corresponds to "Component"; schematic uses definition to make a "manifest"                                  | Also known as "template"
+| Slot               | Schematic calls these "attributes"; the range can be typical types (string, int, etc.) or enumerations (below) | Also known as "property"
+| Enum (enumeration) | Schematic calls these "valid values"                                                                                | Also known as "controlled values"  
+
+Classes depend on slots being defined, and some slots depend on enumerated values being defined. 
+If a class uses a slot that is not defined, the model will error when trying to build; and same with a slot that uses an enumeration.
+Slots can be reused across classes; and same with enumerations for slot. 
+For example, `unit` can be reused across any class entities that needs to capture unit information.
+
+#### Classes
+
+Classes have slots (properties). 
+All classes are grouped under `modules/Template`. 
+Classes can be built upon, so subclasses inherit properties from a parent class. 
+
+##### Example: Base patient class
+
+Here is an small-ish base class definition for a patient:
+
+```
+  PatientTemplate:
+    is_a: Template
+    description: >
+      Template for collecting *minimal* individual-level patient data. 
+    slots:
+    - individualID
+    - sex
+    - age
+    - ageUnit
+    - species # should be constrained to human
+    - diagnosis
+    - nf1Genotype
+    - nf2Genotype
+    annotations:
+      requiresComponent: ''
+      required: false
+```
+
+#### Slots
+
+Slots have a range, which can be a basic data type such as "string" and "integer", or a set of controlled values (see Enum). 
+If the range of a slot is not explicit defined, it defaults to "string" (basically free text).
+All slots are in the file `modules/props.yaml`. 
+
+##### Example slots
+
+```
+slots:
+  #...more above
+  tissue:
+    description: Association with some tissue (a mereologically maximal collection of cells that together perform physiological function).
+    range: TissueEnum # Take a look at TissueEnum below
+    required: false
+  title:
+    description: Title of a resource.
+    meaning: https://www.w3.org/TR/vocab-dcat-3/#Property:resource_title
+    required: true
+  # ...more below
+```
+
+Here, `required` defines globally whether this slot *must* be filled out.
```

Comment:
perhaps here we can add if you need the case where it's not required globally what to do (if there is a solution)

Reply:
That's a good suggestion